.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
All files located in .vscode/ folder should be in the .gitignore and not be commit. Please remove this file from version control